src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1173,7 +1049,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         {
             try
             {
-                auto acrylicBrush = _root.Background().as<Media::AcrylicBrush>();
+                auto acrylicBrush = RootGrid().Background().as<Media::AcrylicBrush>();

```

Comment:
Just curious, do you know why `root` instead of `*this`?

Reply:
You should almost certainly read the preceding comments and the second commit in this PR :wink: